project/Dependencies.scala

```diff
@@ -41,10 +33,5 @@ object Dependencies {
     commonsLang, // for levenshtein distance impl
     akkaRemoting % Provided,
     akkaClusterMetrics % Provided,
-    management % Provided,
```

Comment:
Strange that a provided dependency would change the dependency downstream like that, but good to cleanup this and have as few dependencies as possible.

It probably had a http route for Akka Management previously, that we don't include now.